financial-connections/src/test/java/com/stripe/android/financialconnections/screenshottests/FinancialConnectionsShotShowkaseScreenshotTest.kt

```diff
@@ -57,12 +65,19 @@ class PaparazziSampleScreenshotTest {
             )
         )
         paparazzi.snapshot {
-            CompositionLocalProvider(
-                LocalInspectionMode provides true,
-                LocalTopAppBarHost provides FakeTopAppBarHost(),
-            ) {
-                Box {
-                    componentTestPreview.Content()
+            FinancialConnectionsTheme {
```

Comment:
Making the screenshots only cover the actual content. If you prefer, I can add the top app bar back in here.

Reply:
Don't have a strong preference, but maybe having the entire scaffold with app bar helps catching top padding regressions, maybe app bar elevation related UI details, etc